cmd/query/app/grpc_handler.go

```diff
@@ -89,10 +89,12 @@ func (g *GRPCHandler) GetTrace(r *api_v2.GetTraceRequest, stream api_v2.QuerySer
 	if r.TraceID == (model.TraceID{}) {
 		return errUninitializedTraceID
 	}
-	query := spanstore.GetTraceParameters{
-		TraceID:   r.TraceID,
-		StartTime: r.StartTime,
-		EndTime:   r.EndTime,
+	query := querysvc.GetTraceParameters{
```

Comment:
I recommend adding Raw field to the new types and populating it in this PR, because here you have explicit places (due to compile errors) where you need to make changes and the raw parameter is generally already available from the inputs.

Reply:
@yurishkuro done